qemu/tests/cfg/qemu_iotests.cfg

```diff
@@ -1,27 +1,31 @@
 - qemu_iotests:
+    only Linux
```

Comment:
I think we need the parameter. as "only Linux" is used for guest and " only Host_RHEL.m7, Host_RHEL.m6" is set for host